network/ovs_endpointclient_linux.go

```diff
@@ -154,16 +152,10 @@ func (client *OVSEndpointClient) AddEndpointRules(epInfo *EndpointInfo) error {
 
 func (client *OVSEndpointClient) DeleteEndpointRules(ep *endpoint) {
 	log.Printf("[ovs] Get ovs port for interface %v.", ep.HostIfName)
-	containerPort, err := client.ovsctlClient.GetOVSPortNumber(client.hostVethName)
-	if err != nil {
-		log.Printf("[ovs] Get portnum failed with error %v", err)
-	}
+	containerPort := ep.VethOVSSwitchPort
 
 	log.Printf("[ovs] Get ovs port for interface %v.", client.hostPrimaryIfName)
-	hostPort, err := client.ovsctlClient.GetOVSPortNumber(client.hostPrimaryIfName)
-	if err != nil {
-		log.Printf("[ovs] Get portnum failed with error %v", err)
-	}
+	hostPort := ep.VethOVSSwitchPort
```

Comment:
this should be extIf.OvsSwitchPort

Reply:
same as above - this func is not getting extIf passed anywhere, it is passing ep endpoint struct, do we need to pass * network struct to this function caller here?

Reply:
yes..newovsendpointclient pass nw field, we should save in ovsendpointclient structure

Reply:
added